config/locales/blacklight.en.yml

```diff
@@ -183,6 +183,8 @@ en:
         counter: '%{counter}. '
       facets:
         title: 'Limit your search'
+        filter: 'Filter'
+        clear: 'Clear'
```

Comment:
Before merging this, we should get translations for our other languages. It might be as easy as pulling equivalent values out of other fields..